python/multicorn/utils.py

```diff
@@ -21,12 +21,13 @@ def _log_to_postgres(message, level=0, hint=None, detail=None):
 
 
 class MulticornException(Exception):
-    def __init__(self, message, code, hint, detail):
+    def __init__(self, message, code, hint, detail, sqlstate=None):
```

Comment:
Shouldn't it be mandatory the sqlstate? Why low level ones like `detail`, `hint` are mandatory, and the actual _error code_ isn't. It feels a bit like it's going to be a door open to just not bothering.

Reply:
Or maybe internally the main usage to blindly propagate strings without knowing, except in few cases where the exception will be crafted with some care.